wasmbinding/stargate_bindings.go

```diff
@@ -0,0 +1,76 @@
+package wasmbinding
+
+import (
+	"sync"
+
+	"github.com/osmosis-labs/osmosis/v10/wasmbinding/stargate/auth"
+	"github.com/osmosis-labs/osmosis/v10/wasmbinding/stargate/bank"
+	distr "github.com/osmosis-labs/osmosis/v10/wasmbinding/stargate/distribution"
+	"github.com/osmosis-labs/osmosis/v10/wasmbinding/stargate/feegrant"
+	"github.com/osmosis-labs/osmosis/v10/wasmbinding/stargate/mint"
+	"github.com/osmosis-labs/osmosis/v10/wasmbinding/stargate/slashing"
+	"github.com/osmosis-labs/osmosis/v10/wasmbinding/stargate/staking"
+)
+
+// StargateLayerBindings keeps whitelist and its deterministic
+// response binding for stargate queries.
+//
+// The query can be multi-thread, so we have to use
+// thread safe sync.Map instead map[string]bool.
+var StargateLayerBindings sync.Map
+
+func init() {
+	// auth
+	StargateLayerBindings.Store("/cosmos.auth.v1beta1.Query/Account", &auth.QueryAccountResponse{})
```

Comment:
I'm wondering if we can use these protos directly from `github.com/cosmos/cosmos-sdk` packages instead of defining them within our repo

Reply:
I think we should be using them directly from the sdk. 

In https://github.com/osmosis-labs/wasmd/pull/7/files, I noticed that there is a function called "NormalizeResponses".

IMO, there is no need for it if we are importing these from the SDK